spec/adapters/base_job_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require "active_job"
+
+RSpec.describe GraphQL::Adapters::BaseJob, type: :job do
+  ActiveJob::Base.queue_adapter = :inline
```

Comment:
Never define configuration outside of `before` and `after` hooks, and always restore the value after each test. Otherwise it's easier to introduce unpredictable global state and flakiness.

Reply:
Thank you for the advice!